File: TouchPOS_API/Pay_True/TrueRequestModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TouchPOS_API.Pay_True
{
    public class TrueRequestModel
    {

        public string isv_payment_ref { get; set; }
        public string merchant_id { get; set; }
        public string currency { get; set; }
        public string request_amount { get; set; }
        public string payment_method { get; set; }
        public string payment_code { get; set; }
        public string description { get; set; }
        public  MetaData  metadata { get; set; }

        public class MetaData
        {
            public string partner_shop_id { get; set; }
        }
    }
}

File: TouchPOS/Payment/Pay_true/TrueRequestModel.vb
Public Class TrueRequestModel
    Public isv_payment_ref As String
    Public merchant_id As String
    Public currency As String
    Public request_amount As String
    Public payment_method As String
    Public payment_code As String
    Public description As String
    Public metadata As New MetaData


End Class
Public Class MetaData
    Public shop_id As String ' ของทรู
    Public partner_shop_id As String
    Public partner_shop_name As String
End Class

